app/django_setup.py

```diff
@@ -51,24 +49,50 @@
 django.conf.settings.USE_L10N = True
 django.conf.settings.LOCALE_PATHS = ('locale',)
 django.conf.settings.LANGUAGES_BIDI = LANGUAGES_BIDI
-django.conf.settings.TEMPLATE_LOADERS = ('django_setup.TemplateLoader',)
 
+# https://docs.djangoproject.com/en/1.9/ref/templates/upgrading/#the-templates-settings
+django.conf.settings.TEMPLATES = [
+    {
+        'BACKEND': 'django.template.backends.django.DjangoTemplates',
+        'DIRS': [
+        ],
+        'OPTIONS': {
+            'context_processors': [
+                'django.contrib.auth.context_processors.auth',
+                'django.template.context_processors.debug',
+                'django.template.context_processors.i18n',
+                'django.template.context_processors.media',
+                'django.template.context_processors.static',
+                'django.template.context_processors.tz',
+                'django.contrib.messages.context_processors.messages',
+            ],
+            'loaders': [
+                'django_setup.TemplateLoader',
+            ]
+        },
+    },
+]
 
-class TemplateLoader(django.template.loader.BaseLoader):
+# It's required to call this function when we use Django features outside
+# Django framework.
+django.setup()
+
+class TemplateLoader(django.template.loaders.base.Loader):
     """Our custom template loader, which loads templates from Resources."""
-    is_usable = True  # Django requires this flag
 
-    def load_template(self, name, dirs):
```

Comment:
Yes, it's required to override `get_contents()`. Otherwise it's considered as an old format:
https://github.com/django/django/blob/master/django/template/loaders/base.py#L104

It looks usually preferred to override fine-grained methods `get_contents()` and `get_template_sources()`. But it looks more effort is needed to adopt our logic to the two methods. So I overrode `get_template()` for now.